### DISCUSSION TITLE
Allow dynamic prompts in ADetailer

### DISCUSSION THREAD

**Discussion by MancV21** [adieyal/sd-dynamic-prompts#548] [Ideas]
Hi. I've looked around to see if this is already possible and can't find it. I'd like to use this extension with [ADetailer](https://github.com/Bing-su/adetailer), so I can generate images with different faces in the one image. I've also asked the dev of ADetailer to implement dynamic prompts with his extension in case the issue is more with that extension than this. If it is possible and I've missed it, can anyone tell me how? Thanks.

**Comment by llotenberg** [comment]
Have you actually tried this yet? Because as far as I can tell, it should just work.

**Reply by MancV21** [reply]
I have yes. It turns out it IS working but the output from AUTOMATIC1111 wasn't showing it correctly, except for in the console. :)

**Comment by adieyal** [comment]
awesome - I didn't think about trying it.
